auction-app-backend/src/main/resources/application.properties

```diff
@@ -1,2 +1,20 @@
+# Swagger config
 spring.mvc.pathmatch.matching-strategy = ANT_PATH_MATCHER
-API_PREFIX=/api/v1
+
+# API config
+application.api.prefix=/api/v1
+
+# DB config
+spring.datasource.url=jdbc:postgresql://localhost:5432/auctionapp
+spring.datasource.username=
+spring.datasource.password=
+spring.jpa.hibernate.ddl-auto=update
+spring.jpa.show-sql=true
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.properties.hibernate.format_sql=true
+
+# JWT config
+application.jwt.secretKey=itsnotmuchofasecretattallunfortunatelly
```

Comment:
This value should not be visible on GitHub. 

For development purposes, you can create separate file (I suggest putting it outside of project folder) which will contain _sensitive_ data, like this and use that file when running the app (this way you won't need revert changes on sensitive data before each commit of this file). This file (`application.properties`) should be maintained in a way it contains all properties necessary for running the application, but the value of properties does not have to be specified (and value of _sensitive_ properties must not be specified).

Reply:
OK. We probably need to discuss this a little bit more.